src/databases/Silo/avtSiloFileFormat.C

```diff
@@ -6895,6 +6896,7 @@ PaintNodesForAnnotIntFacelist(vtkBitArray *nlvar,
                         edge[2][0] = zl->nodelist[nlIdx+2];
                         edge[2][1] = zl->nodelist[nlIdx+0];
                         nedges = 3;
+                        break;
```

Comment:
do these break statements change the functionality?

Reply:
Yes. Certainly. There is no question fall-through is NOT intended here. Each case is for a wholly different situation. The cases do NOT build upon each other. They are exclusive. Now, if you asked me to try to explain how this code has been here this long in this state...I can't. Except that annotation-int objects are not too terribly common and if they are always meshes of eaither quads or hexs, then only that (last) case in either switch was getting triggered. I think that is entirely possible.

Reply:
That makes sense. Thanks for investigating for me.